haystack/nodes/prompt/prompt_node.py

```diff
@@ -561,6 +626,18 @@ def __init__(
         use_gpu: Optional[bool] = None,
         devices: Optional[List[Union[str, torch.device]]] = None,
     ):
+        """
+        Creates a PromptNode instance.
+
+        :param model_name_or_path: The name of the model to use or an instance of PromptModel.
```

Comment:
Could we explain here or in the class doc string above what kinds of models are supported for the `PromptNode`? For example, I think we can only use `Seq2Seq` models from HuggingFace transformers, right?

Reply:
We can, but we don't want to tell them (I think). See https://github.com/deepset-ai/haystack/pull/3821#discussion_r1064398086